lib/planet-str.lua

```diff
@@ -0,0 +1,31 @@
+local lib=require("lib.lib")
+
+local ps={}
```

Comment:
Please use the convention of calling this Public. This means we don't need to rename functions if we move them to a different file